src/clarity-angular/forms/_variables.select.scss

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+// Usage: ../forms/_select.clarity.scss
+$select-hover-background: rgba(221, 221, 221, 0.5);
```

Comment:
Should we prefix these variables with `clr`?

Reply:
yes. Will update.

Reply:
Changed the name's